src/WormholeSender.sol

```diff
@@ -24,4 +24,10 @@ abstract contract WormholeSender is WormholeBase {
   function quoteDeliveryCost(uint16 targetChain) public view virtual returns (uint256 cost) {
     (cost,) = WORMHOLE_RELAYER.quoteEVMDeliveryPrice(targetChain, 0, GAS_LIMIT);
   }
+
+  /// @param gasLimit The new gas limit value which is used to estimate the delivery cost for
+  /// sending messages cross chain.
+  function updateGasLimit(uint256 gasLimit) public onlyOwner {
+    GAS_LIMIT = gasLimit;
```

Comment:
let's add an event and make the param `_gasLimit`

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/94/commits/5fb57cdad6d3b0bce2cf8f63c6c873ef27f761ae